src/main/java/milutrock/MiluTrock.java

```diff
@@ -4,6 +4,10 @@
 
 import milutrock.exceptions.UnknownCommandException;
 
+/**
+ * A task management program that interacts with users through a
+ * command-line interface.
+ */
 public class MiluTrock {
```

Comment:
The class name `MiluTrock` seems unique but does not convey any meaningful information about what the class does. A more descriptive name could enhance the readability and maintainability of the code. For example, if this class is responsible for managing tasks, a name like `TaskManager` might be more appropriate.